src/components/NotFound/NotFound.css

```diff
@@ -0,0 +1,28 @@
+body {
+  background-size: cover;
+  background-image: radial-gradient(
```

Comment:
This is a fun gradient, glad you are still playing around with new CSS properties!